test/cpp/end2end/end2end_test.cc

```diff
@@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
   explicit TestServiceImpl(const grpc::string& host)
```

Comment:
I think this is moved to its own file and thus you need a merge. Sorry about that...